packages/web/modals/validator-next-step.tsx

```diff
@@ -60,6 +63,10 @@ export const ValidatorNextStepModal: FunctionComponent<
               className="w-full"
               mode="primary-bullish"
               onClick={() => {
+                logEvent([
+                  EventName.Stake.squadOptionClicked,
+                  { option: "keep" },
+                ]);
                 onRequestClose();
                 alert("make stake call now");
```

Comment:
Can do in a sep PR but the stake call is merged so you should be able to put it in here